src/kbc_modules/cc_kbc/attester/sample/mod.rs

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) 2022 Alibaba Cloud
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+use super::Attester;
+use anyhow::*;
+use serde::{Deserialize, Serialize};
+use std::env;
+
+// If the environment variable "AA_SAMPLE_ATTESTER_TEST" is set,
+// the TEE platform is considered as "sample".
+pub fn detect_platform() -> bool {
+    env::var("AA_SAMPLE_ATTESTER_TEST").is_ok()
```

Comment:
If setting this env-var causes the behaviour of this component to become a NOP, that worries me: could this be a way to compromise the security of the system?

Also, the name of this variable is a bit ambiguous as "_TEST" might imply this functionality is simply added for testing. Is that the case, or is this just the variable used "to test" if we're running with a sample TEE?